lstchain/image/muon/muon_analysis.py

```diff
@@ -291,3 +342,98 @@ def tag_pix_thr(image, thr_low = 50, thr_up = 500, pe_thr = 10):
     """
 
     return ((np.size(image[image > pe_thr]) < thr_up) and (np.size(image[image > pe_thr]) > thr_low))
+
+
+def radial_light_distribution(centre_x, centre_y, pixel_x, pixel_y, image):
+    """
+
+    Parameters
+    ----------
+    centre_x: float
+        Centre of muon ring in the field of view from circle fitting
+    centre_y: float
+        Centre of muon ring in the field of view from circle fitting
+    pixel_x: ndarray
+        X position of pixels in image
+    pixel_y: ndarray
+        Y position of pixel in image
+    image: ndarray
+        Amplitude of image pixels
+
+    Returns
+    -------
+    standard_dev, skewness
+    """
+    # Convert everything to degrees:
+    x0 = centre_x.to(u.deg).value
+    y0 = centre_y.to(u.deg).value
+    pix_x = pixel_x.to(u.deg).value
+    pix_y = pixel_y.to(u.deg).value
+
+    pix_r = np.hypot(pix_x-x0, pix_y-y0)
```

Comment:
hypot is pretty slow as it does all sorts of edge case detection for very small  float values. If you don't expect this to happen, just use `np.sqrt((pix_x - x0)**2 + (pix_y - y0)**2)`

Reply:
Done, thx